### ISSUE TITLE
Android app stops working

### ISSUE THREAD

**Issue by JulianvanEck** [paypal/PayPal-Cordova-Plugin#59]
Hi. I am experiencing an issue on Android devices where the app stops working as soon as I try to render the paypal UI.

**Comment by braebot** [comment]
Hi there!  Can you please provide more information?  Some screenshots, sdk versions, and log traces would be very useful!  Thanks!

**Comment by omaisali** [comment]
I have same issue with Cordova . 
App stops when click on Paypal button.

**Comment by romk1n** [comment]
@JulianvanEck @omaisali in addition to what @braebot has asked can you also see if you can reproduce the same behavior with the sample app [here](https://github.com/romk1n/MyCordovaShop) Thanks.

**Comment by JulianvanEck** [comment]
@braebot @romk1n I am using Sencha Touch 2.3 with this plugin. I have basicly used the sample code exactly as it is and added the following in my config.xml:
<gap:plugin name="com.paypal.cordova.mobilesdk" source="plugins.cordova.io" version="3.1.4" />
I also added the following to my index.html file:
 <script type="text/javascript" src="resources/js/paypal-mobile-js-helper.js"></script>
I make use of the phonegap build service, so do I need to include the sdk files in the project as well? The same thing was happening on iPad but after changing to version 3.1.4 the problem was solved on the iPad. Thanks for the assistance in advance.
